metadata-ingestion/src/datahub/ingestion/source/schema/json_schema.py

```diff
@@ -0,0 +1,364 @@
+            platform=self.config.platform,
+            name=dataset_name,
+            platform_instance=self.config.platform_instance,
+        )
+        yield self._report_and_return(
+            MetadataChangeProposalWrapper(
+                entityUrn=dataset_urn, aspect=meta
+            ).as_workunit()
+        )
+
+        # status
+        yield self._report_and_return(
+            MetadataChangeProposalWrapper(
+                entityUrn=dataset_urn, aspect=models.StatusClass(removed=False)
+            ).as_workunit()
+        )
+
+        # datasetProperties
+        yield self._report_and_return(
+            MetadataChangeProposalWrapper(
+                entityUrn=dataset_urn,
+                aspect=models.DatasetPropertiesClass(
+                    externalUrl=JsonSchemaTranslator._get_id_from_any_schema(
+                        schema_dict
+                    ),
+                    name=dataset_simple_name,
+                    description=JsonSchemaTranslator._get_description_from_any_schema(
+                        schema_dict
+                    ),
+                ),
+            ).as_workunit()
+        )
+
+        # subTypes
+        yield self._report_and_return(
+            MetadataChangeProposalWrapper(
+                entityUrn=dataset_urn, aspect=models.SubTypesClass(typeNames=["schema"])
+            ).as_workunit()
+        )
+
+        browse_path = browse_prefix + dirname(schema_type)
+        yield self._report_and_return(
+            MetadataChangeProposalWrapper(
+                entityUrn=dataset_urn,
+                aspect=models.BrowsePathsClass(paths=[browse_path]),
+            ).as_workunit()
+        )
+
+        if self.config.platform_instance:
+            yield self._report_and_return(
+                MetadataChangeProposalWrapper(
+                    entityUrn=dataset_urn,
+                    aspect=models.DataPlatformInstanceClass(
+                        platform=str(
+                            DataPlatformUrn.create_from_id(self.config.platform)
+                        ),
+                        instance=make_dataplatform_instance_urn(
+                            platform=self.config.platform,
+                            instance=self.config.platform_instance,
+                        ),
+                    ),
+                ).as_workunit()
+            )
+
+        self.stale_entity_removal_handler.add_entity_to_state(type="_", urn=dataset_urn)
+
+    def get_workunits(self) -> Iterable[Union[MetadataWorkUnit, UsageStatsWorkUnit]]:
+        if self.config.uri_replace_pattern:
+            ref_loader = functools.partial(
+                self.stringreplaceloader,
+                self.config.uri_replace_pattern.match,
+                self.config.uri_replace_pattern.replace,
+            )
+        else:
+            ref_loader = jsonref.jsonloader
+        browse_prefix = f"/{self.config.env.lower()}/{self.config.platform}"
```

Comment:
Nitpick: We are hoping to eradicate "env" - thoughts on keeping it out of this in favor of platform instance